templates/contacts/contact_invite_participants.haml

```diff
@@ -0,0 +1,487 @@
+                                  %span.sort.icon.icon-arrow-down-2.sort-asc.visible
+                            -else
+                              %a{href:"?sort_on=-{{ field.uuid }}{% if search %}&search={{ search|escape }}{% endif %}"}
+                                %div
+                                  {{ field.label }}
+                                  %span.sort.icon.icon-arrow-up5.sort-desc.visible
+                          -else
+                            %a{href:"?sort_on=-{{ field.uuid }}{% if search %}&search={{ search|escape }}{% endif %}"}
+                              %div
+                                {{ field.label }}
+                                %span.sort.icon.icon-arrow-down-2.sort-desc
+                    %th
+                      - if object_list
+
+                        -if sort_field == 'created_on'
+                          -if sort_direction == 'desc'
+                            %a{href:"?sort_on=created_on{% if search %}&search={{ search|escape }}{% endif %}"}
+                              %div
+                                -trans "Created On"
+                                %span.sort.icon.icon-arrow-down-2.sort-asc.visible
+                          -else
+                            %a{href:"?sort_on=-created_on{% if search %}&search={{ search|escape }}{% endif %}"}
+                              %div
+                                -trans "Created On"
+                                %span.sort.icon.icon-arrow-up5.sort-desc.visible
+                        -else
+                          %a{href:"?sort_on=-created_on{% if search %}&search={{ search|escape }}{% endif %}"}
+                            %div
+                              -trans "Created On"
+                              %span.sort.icon.icon-arrow-down-2.sort-desc
+                      %th
+
+                  %tbody
+                    - for object in contacts
+                      %tr.contact.select-row.object-row{'class':'{% cycle row1 row2 %}', 'data-uuid': '{{object.uuid}}', 'data-object-id': '{{object.id}}'}
+
+                        %td.value-text.field_name
+                          {{ object|name:user_org }}
+
+                        %td.value-phone.field_phone
+                          %nobr
+                            {{ object|urn:user_org }}
+
+                        -for field in contact_fields
+                          -if field.show_in_table
+                            %td.field
+                              {{ object|contact_field:field.key }}
+
+                        %td.value-received.field_received
+                          %nobr
+                            {% short_datetime object.created_on %}
+                        
+                        %td.invite-column.value-received.field_received
+                          %a.invite.btn.btn-primary.btn-block{href:"{% url 'contacts.contact_invite_participants' %}", 'data-omnibox': "c-{{object.uuid}}"}
```

Comment:
The invite button is not visible

![Screen Shot 2020-04-14 at 18 02 36](https://user-images.githubusercontent.com/5198870/79273932-23cd7a00-7e7a-11ea-800a-cdd8e8481150.png)

Reply:
Changed 'Invite' link to always display.